index.js

```diff
@@ -0,0 +1,192 @@
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const inquirer = require("inquirer");
+const fs = require("fs");
+const generateTeam = require("./src/generateTeam");
+const { ensureExpectedIsNonNegativeInteger } = require("jest-matcher-utils");
+
+team = [];
+
+const managerQuestions = () => {
+  inquirer
```

Comment:
for readability, consider placing only your questions inside of the managerQuestions variable and leverage the inquirer logic elsewhere, or change the name of the function to promptManagerQuestions. 
By the way, fantastic to see you implementing arrow functions, great stuff!! :-)